modular_skyrat/modules/primitive_fun/code/big_mortar.dm

```diff
@@ -0,0 +1,115 @@
+/obj/structure/large_mortar
+	name = "large mortar"
+	desc = "A large bowl perfect for grinding and or juicing a large number of things at once."
+	icon = 'modular_skyrat/modules/primitive_fun/icons/cooking_structures.dmi'
+	icon_state = "big_mortar"
+	density = TRUE
+	anchored = TRUE
+	max_integrity = 100
+	pass_flags = PASSTABLE
+	custom_materials = list(/datum/material/wood = MINERAL_MATERIAL_AMOUNT * 10)
+	/// All of the items stored within this structure
+	var/list/contained_items
+	/// The maximum number of items this structure can store
+	var/maximum_contained_items = 5
```

Comment:
I just *know* someone is going to say make this a define-- I will spare you now and just tell you "make this a define."

Reply:
This is a case where it doesn't need to be a define and nobody's gonna say make it one, because if you want to change it at all its right there at the variable.

It'd need to be a define /maybe/ if it wasn't a var and instead was just harcoded.

Reply:
I'm saying the 5, and yes, someone will say it (Zone, Golden, etc.)

Reply:
Ah damn it, just ignore me, hopefully I'm wrong.